docs(tflite): remove '--linkopt' flag

CoreyCole opened the pull request:
In the comments above these two TFLite GPU delegate build configurations, the suggested command threw an error for me on both OSX and Ubuntu with various gcc/clang versions.

See issue https://github.com/tensorflow/tensorflow/issues/44926

CoreyCole committed:
docs(tflite): remove '--linkopt' flag

In the comments above these two TFLite GPU delegate build configurations, the suggested command threw an error for me on both OSX and Ubuntu with various gcc/clang versions.

See issue https://github.com/tensorflow/tensorflow/issues/44926

jdduke commented on tensorflow/lite/delegates/gpu/BUILD:
```diff
@@ -116,7 +116,7 @@ objc_library(
     alwayslink = 1,
 )
 
-# build -c opt --config android_arm64 --copt -Os --copt -DTFLITE_GPU_BINARY_RELEASE --copt --linkopt -s --strip always :libtensorflowlite_gpu_gl.so
```
I think it should actually be the preceding `1--copt` which is removed.  That is, it should be

```
--copt -DTFLITE_GPU_BINARY_RELEASE --linkopt -s
```

CoreyCole replied:
fixed in https://github.com/tensorflow/tensorflow/pull/45049/commits/54e6c38245185fe9e4eb6ad38b370d9b14562e97

CoreyCole committed:
docs(tflite): add bacl --linkopt flag

remove --copt preceding the --linkopt
https://github.com/tensorflow/tensorflow/pull/45049/files#r556732278

jdduke commented on tensorflow/lite/delegates/gpu/BUILD:
```diff
@@ -141,7 +141,7 @@ cc_binary(
 )
 
-# build -c opt --config android_arm64 --copt -Os --copt -DTFLITE_GPU_BINARY_RELEASE --copt --linkopt -s --strip always :libtensorflowlite_gpu_delegate.so
+# build -c opt --config android_arm64 --copt -Os --copt -DTFLITE_GPU_BINARY_RELEASE --copt -s --strip always :libtensorflowlite_gpu_delegate.so
```
Same with this line, sorry should have flagged earlier.

CoreyCole replied:
ah that is my bad--I haven't looked at this file in a while. Fixed in https://github.com/tensorflow/tensorflow/pull/45049/commits/4914abba00cb5b278d9cf3b7ef0c2345357a72ed

CoreyCole committed:
add back --linkopt flag
